src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -43,8 +53,16 @@ public ClimbSubsystem() {
     leadClimbRotate = new WPI_TalonSRX(LEAD_ROTATE_MOTOR);
     followClimbRotate = new WPI_TalonSRX(FOLLOW_ROTATE_MOTOR);
 
+    pulse = new SensorCollection(leadClimbRotate);
```

Comment:
is this the preferred way to instantiate these? i think this should be the following

```java
pulse = leadClimbRotate.getSensorCollection();
```

not sure if thats the correct method but same idea